shared/structures/src/SetupSteps.ts

```diff
@@ -6,9 +6,12 @@ export enum SetupStepType {
     Groups = "Groups",
     Premises = "Premises",
     Emails = 'Emails',
-    Payment = 'Payment'
+    Payment = 'Payment',
+    Registrations = 'Registrations',
 }
 
+export const minimumRegistrationCount: number = 1;
```

Comment:
Kan jij via Slack even vragen of dit hoger moet zijn dan 1, of instelbaar moet zijn per standaard leeftijdsgroep?